### ISSUE TITLE
getStats() localAudioTrackStats returning audioLevel = null

### ISSUE THREAD

**Issue by bryanhpchiang** [twilio/twilio-video.js#1140]
- [x ] I have verified that the issue occurs with the latest twilio-video.js release and is not marked as a known issue in the [CHANGELOG.md](https://github.com/twilio/twilio-video.js/blob/master/CHANGELOG.md).
 - [x ] I reviewed the [Common Issues](https://github.com/twilio/twilio-video.js/blob/master/COMMON_ISSUES.md) and open GitHub issues and verified that this report represents a potentially new issue.
 - [ x] I verified that the [Quickstart](https://github.com/twilio/video-quickstart-js) application works in my environment.
 - [ x] I am not sharing any [Personally Identifiable Information (PII)](https://www.twilio.com/docs/glossary/what-is-personally-identifiable-information-pii)
       or sensitive account information (API keys, credentials, etc.) when reporting this issue.

**Code to reproduce the issue:**

```js
// TODO

  if (room.current) {
        room.current.getStats().then((report) => {
          console.log(report);
          if (report.length > 0) {
            report[0].remoteAudioTrackStats.forEach((stat) => {
              console.log(stat.audioLevel, stat.trackId);
            });
          }
        });
      }
```

**Expected behavior:**

When I'm talking, the first track in report.localAudioTrackStats should have an audioLevel that is not null

**Actual behavior:**

, but right now it is always null. The audio levels are showing up fine for remoteAudioTrackStats.

TODO

**Software versions:**

 - [ ] Browser(s): Latest Chrome
 - [ ] Operating System: macOS 
 - [ ] twilio-video.js:
 - [ ] Third-party libraries (e.g., Angular, React, etc.): React

**Comment by makarandp0** [comment]
Thank you for reporting this issue @bryanhpchiang. We do have a bug tracking this issue (JSDK-2842) in our backlog. We should get to it soon. I will update you once it gets fixed.

**Comment by bryanhpchiang** [comment]
Thanks for the update!

**Comment by dambusm** [comment]
Hi, I am experiencing this issue still in 2.12.0. Could this issue be reopened?

**Comment by charliesantos** [comment]
Should be fixed in 2.18.3. However, audioLevel is not supported in firefox, and in local audio tracks in safari.
